editor/src/components/canvas/canvas-strategies/strategies/reparent-strategy-helpers.ts

```diff
@@ -367,22 +369,29 @@ export function getReparentTargetUnified(
 
   // if the mouse is over the canvas, return the canvas root as the target path
 
-  const flexElementsUnderPoint = [...filteredElementsUnderPoint]
-    .reverse()
-    .filter((element) =>
-      MetadataUtils.isFlexLayoutedContainer(
-        MetadataUtils.findElementByElementPath(metadata, element),
-      ),
-    )
+  const staticContainersUnderPoint = mapDropNulls((element) => {
```

Comment:
I'm a bit confused by this name, without reading the direction helper I would expect it to handle only flow layouts

Reply:
I tried to use the terminology that static == flex or flow
Isn't that a good idea?

Reply:
I think my confusion comes from `position: static`, but I don't have any good suggestion :(

Reply:
renamed!